CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 * Hide the suggestion help from the relationship input list when the user starts typing a search term.
 
+### Adds
+
+* Adds `touch` task for every piece type. It'll perform a database update on all non-archived records of that type trigger the registered for that type relevant events. Example usage: `node app article:touch`.
```

Comment:
Middle sentence is confusing. How about:

"This task invokes `update` on each piece, which will execute all of the same event handlers that normally execute when a piece of that type is updated."